public/datasources.json

```diff
@@ -1,70 +1,102 @@
+    {"name":"Rural Bank","url":"https://api.cdr.ruralbank.com.au/cds-au/v1"},
+    {"name":"Service One Alliance Bank","url":"https://api.cdr.serviceone.com.au/cds-au/v1"},
+    {"name":"South West Credit Union","url":"https://internetbanking.swcredit.com.au/OpenBanking/cds-au/v1"},
+    {"name":"South West Slopes Credit Union","url":"https://online.swscu.com.au/OpenBanking/cds-au/v1"},
+    {"name":"Southern Cross Credit Union","url":"https://mvp1.sccu.com.au/OpenBanking/cds-au/v1"},
+    {"name":"Summerland Credit Union","url":"https://public.cdr-api.summerland.com.au/cds-au/v1"},
+    {"name":"Suncorp","url":"https://api-pub-cdr.suncorpbank.com.au/cds-au/v1","icon":"https://suncorp.com.au/content/dam/suncorp/corporate/images/logos/Suncorp_New_Logo.png"},
+    {"name":"Sydney Mutual Bank","url":"https://internetbanking.sydneymutual.bank/openbankingscu/cds-au/v1"},
+    {"name":"Teachers Mutual Bank","url":"https://ob.tmbl.com.au/tmbank/cds-au/v1"},
+    {"name":"The Capricornian Ltd","url":"https://onlinebanking.capricornian.com.au/OpenBanking/cds-au/v1","icon":"https://www.capricornian.com.au/wp-content/themes/capricornian/images/capricornian-logo.png"},
+    {"name":"The Mac Credit Union","url":"https://onlinebanking.themaccu.com.au/OpenBanking/cds-au/v1"},
+    {"name":"Traditional Credit Union Limited","url":"https://prd.tcu.com.au/cds-au/v1","icon":"https://tcu.com.au/wp-content/uploads/2020/09/TCU_logo_icon.png"},
+    {"name":"Transport Mutual Credit Union","url":"https://banking.transportmutual.com.au/OpenBanking/cds-au/v1"},
+    {"name":"UBank","url":"https://openbank.api.ubank.com.au/cds-au/v1","icon":"https://www.ubank.com.au/content/dam/ubank/img/ubank-logo-green.png"},
+    {"name":"Unity Bank","url":"https://ibanking.unitybank.com.au/openbanking/cds-au/v1","icon":"https://asset.unitybank.com.au/images/Unity-logo-335x115.png"},
+    {"name":"Up","url":"https://api.up.com.au/cds-au/v1","icon":"https://up.com.au/assets/images/logo_400x400.png"},
+    {"name":"Virgin Money Australia","url":"https://secure.api.virginmoney.com.au/cds-au/v1","icon":"https://virginmoney.com.au/content/dam/virginmoney/logos/logo_footer.png"},
+    {"name":"Volt Bank","url":"https://api.voltbank.com.au/cds-au/v1","icon":"https://openbanking.voltbank.com.au/logos/logo?v=1600930338006"},
+    {"name":"Warwick Credit Union","url":"https://onlinebanking.wawcu.com.au/OpenBanking/cds-au/v1"},
+    {"name":"WAW Credit Union","url":"https://onlinebanking.wawcu.com.au/OpenBanking/cds-au/v1"},
+    {"name":"Woolworths Team Bank","url":"https://online.woolworthsteambank.com.au/OpenBanking/cds-au/v1"},
+    {"name":"Xinja Bank","url":"https://cdr.openbank.api.xinja.com/cds-au/v1","icon":"https://xinja.com.au/wp-content/uploads/2019/09/xinja-email-logo-grey-pink-400x96.png"}
```

Comment:
Please remove Xinja. They have discontinued their banking operations in Australia: https://www.itnews.com.au/news/neobank-xinja-hands-back-its-banking-licence-559059